src/index.ts

```diff
@@ -1,21 +1,16 @@
-import puppeteerConfig from './puppeteer-config.json';
 
+export { CompanyTypes, SCRAPERS } from './definitions';
 export { default as createScraper } from './scrapers/factory';
-export { SCRAPERS, CompanyTypes } from './definitions';
 
 // Note: the typo ScaperScrapingResult & ScraperLoginResult (sic) are exported here for backward compatibility
 export {
-  ScraperOptions,
-  ScraperScrapingResult as ScaperScrapingResult,
-  ScraperScrapingResult,
-  ScraperLoginResult as ScaperLoginResult,
-  ScraperLoginResult,
-  Scraper,
-  ScraperCredentials,
+  ScraperLoginResult as ScaperLoginResult, ScraperScrapingResult as ScaperScrapingResult, Scraper,
+  ScraperCredentials, ScraperLoginResult, ScraperOptions, ScraperScrapingResult,
 } from './scrapers/interface';
 
 export { default as OneZeroScraper } from './scrapers/one-zero';
 
 export function getPuppeteerConfig() {
-  return { ...puppeteerConfig };
+  return { chromiumRevision: '1250580' }; // https://github.com/puppeteer/puppeteer/releases/tag/puppeteer-core-v22.5.0
+
 }
```

Comment:
@esakal I reviewed everything, I left the chromium version / scrapers-core to you.